src/frontend/delete-jira-configuration.ts

```diff
@@ -6,26 +6,28 @@ import { Request, Response } from "express";
  * Handle the when a user deletes an entry in the UI
  */
 export default async (req: Request, res: Response): Promise<void> => {
-	const jiraHost = req.session.jiraHost;
+	const jiraHost = req.body.jiraHost;
```

Comment:
Any frontend API calls needs to pass in `jiraHost` as part of body since it wouldn't be set in the session yet.

Reply:
Wouldn't it?

Reply:
No.  this is called from AJAX in the iframe, which cannot set the cookie because of 3rd party cookies being blocked.  This is a more explicit way of doing it.